.github/workflows/build-pdf.yml

```diff
@@ -0,0 +1,60 @@
+name: Build LaTeX document
+# Trigger the workflow on push or pull request
+on: [push, pull_request]
+env:
+  FINAL_FILENAME: Yakoub_13861700_2020.pdf
```

Comment:
Could you just update this default FINAL_FILENAME environment variable to be a more generic thesis.pdf ?

Reply:
Feel free to change it as you wish ;) I allowed edits from maintainers